db/migrate/20230923050924_remove_not_null_on_active_storage_blobs_checksum.active_storage.rb

```diff
@@ -0,0 +1,8 @@
+# This migration comes from active_storage (originally 20211119233751)
+class RemoveNotNullOnActiveStorageBlobsChecksum < ActiveRecord::Migration[6.0]
```

Comment:
These migrations aren't actually necessary since we don't use active storage - maybe I can just remove them